src/entities/Playlists/PlaylistTypes.js

```diff
@@ -0,0 +1,130 @@
+import { uurl } from 'utils/use-url';
+
+class PlaylistTypes {
+  static Echo360ID = 0
+  static YouTubeID = 1
+  static UploadID = 2
+  static KalturaID = 3
+  static MediaSpaceID = 3
+  static BoxID = 4
```

Comment:
```suggestion
  static KalturaID = 3
  static BoxID = 4
```
Doesn't look like `MediaSpaceID` is being used. Is it supposed to still be here?

Reply:
actually it's same as KalturaID, i will remove it